### ISSUE TITLE
Enhance the translate feature

### ISSUE THREAD

**Issue by winstonma** [plateaukao/browser#43]
Love the einkbro on my Onyx Boox Nova

To me, the current translated page is not user friendly. I am not sure if my proposed method would improve the user-friendliness of the result. Please use the desktop browser to test the procedure first.

Let me explain my idea using an example https://stratechery.com/2021/regulators-and-reality/ as an example.

1. Go to the translated URL: https://translate.google.com/translate?hl=en&sl=en&tl=en&u=https://stratechery.com/2021/regulators-and-reality/
2. Open browser's developer console, and then a link will be returned while executing the following command:
```
  document.getElementById("contentframe").children[0].src
```
3. Open the browser with the above link obtained in step 2

I think this way the translated page looks much better. But of coz on step 1 you need to set the correct `sl` and `tl` parameters. I am not sure what is the best way to provide the `sl` and `tl` parameters for the user to select. But the result seems better.

**Comment by winstonma** [comment]
It seems to me that the above steps are overrated. Just go to the link would do.

```
https://stratechery-com.translate.goog/2021/regulators-and-reality/?_x_tr_sl=en&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=ajax,elem
```

It seems to me that there is a general pattern
1. For the domain name, just replace `.` with `-`, then add `.translate.goog`
2. At the end, add `?_x_tr_sl=en&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=ajax,elem`, `_x_tr_sl` is the source language and `_x_tr_tl` is the translated language

**Comment by plateaukao** [comment]
@winstonma Thanks for the suggestion. Yes, it can get better results from the url you provided. I will modify the feature to adopt this way so that it can be more elegant.

Also, I added a button on control bars, so that target language can be selected too, and long press close button to hide all translation controls. :)

**Comment by winstonma** [comment]
@plateaukao really look forward to the new feature 😀

**Comment by winstonma** [comment]
The feature is implemented in v8.15.0 onwards. You could select different translate services by holding the translate button.
